documentation/config_file_master.md

```diff
@@ -12,7 +12,7 @@ canonical: "/puppetserver/latest/config_file_master.html"
 
 The `master.conf` file configures how Puppet Server handles [deprecated][] authorization methods for master endpoints. For an overview, see [Puppet Server Configuration](./configuration.html).
 
-> **Deprecation Note:** This file contains only the `allow-header-cert-info` parameter, and is deprecated as of Puppet Server 2.2 in favor of authorization settings that are configured in the [`auth.conf`][] file. Because this setting is deprecated, a default `master.conf` file is no longer included in the Puppet Server package.
+> **Deprecation Note:** This file contains only the `allow-header-cert-info` parameter, and is deprecated as of Puppet Server 2.2 in favor of authorization settings that are configured in the [new `auth.conf`][] file. Because this setting is deprecated, a default `master.conf` file is no longer included in the Puppet Server package.
```

Comment:
Ahh, I see my issue. The link is to a page that ends with ".html" (which is correct for the docs site) while in github the page is ".md". I was missing the translation that this goes through before it hits the docs site (I went to the branch's file view, and it was a valid link to a 404).